### DISCUSSION TITLE
Welcome to DOOM BFA (Classic-RBDOOM-3-BFG) Discussions!

### DISCUSSION THREAD

**Discussion by MadDeCoDeR** [MadDeCoDeR/Classic-RBDOOM-3-BFG#83] [Announcements]
Hello there.
Welcome to DOOM BFA's GitHub's  Discussions.
These forums will focus mainly on the technical aspects of DOOM BFA (issues, new features, how things are working etc)

**Comment by eljina33** [comment]
Subtitle support for BFG now please

**Reply by MadDeCoDeR** [reply]
But  BFA already supports subtitles (BETA).

**Reply by eljina33** [reply]
there is only the first episode and it was made 2 years ago

**Comment by eljina33** [comment]
there is only the first episode and it was made 2 years ago

**Reply by MadDeCoDeR** [reply]
Yeah but the system to support subtitles is here. The only think that is needed for full fledged subtitles is some one with enough time and willingess to make them. The sample on moddb was an example of how to make them (the nightly builds have some changes)

**Reply by eljina33** [reply]
Will you do this lofty task? I looked and it was very complicated.

**Reply by MadDeCoDeR** [reply]
If I do that it will take ALOT of time, keep in mind that even now I'm still working on the next version of BFA which aim to eliminate as many issues as possible.

**Reply by MadDeCoDeR** [reply]
> Will you do this lofty task? I looked and it was very complicated.

Also what exactly is the complication? Aren't the comments in the file good enough? If there is something that isn't very clear you can tell me

**Reply by eljina33** [reply]
I didn't understand what the numbers on the conversations were

**Reply by MadDeCoDeR** [reply]
Which numbers exactly?

**Reply by eljina33** [reply]
I really want to play with subtitles and learn the story, I tried to do it myself by looking at the subtitle example, but I couldn't, please can you do this sublime task, this game deserves it

**Reply by eljina33** [reply]
it seems very confusing to me

**Reply by eljina33** [reply]
With your knowledge, this job will take a very short time, I really tried to do it myself, but I did not understand anything
